src/go/rpk/pkg/config/config_test.go

```diff
@@ -1941,6 +1956,32 @@ func TestReadFlat(t *testing.T) {
 		RequireClientAuth: true,
 	}}
 
+	conf.Redpanda.AdminApi = []NamedSocketAddress{{
+		SocketAddress: SocketAddress{
+			Address: "192.168.92.34",
+			Port:    9644,
+		},
+		Name: "internal",
+	}, {
+		SocketAddress: SocketAddress{
+			Address: "127.0.0.1",
+			Port:    9645,
+		},
+	}}
```

Comment:
I don't understand when we test `NamedSocketAddress` without name how should the redpanda configuration look like? Maybe I missed the test/assertion with that :)

Reply:
Default listeners doesn't have names. It follows this approche.